### DISCUSSION TITLE
Compatibility with Ionic Framework

### DISCUSSION THREAD

**Discussion by bruno-f-dias** [deta/cloud-docs#732] [Help]
I am working on a small project to help manage some student information. I found out about Deta recently and I’m still reading about your service but I’d like to know if it’s possible to deploy the Ionic project , done in React, along with the database and API. I’m asking because even though Deta is compatible with Node, I’m wondering if there’s any issue with using the Ionic Framework. I’d appreciate it if someone could help with this.

**Comment by sudo-subg** [comment]
Don't know much about Ionic framework but as the docs says it uses web technologies like Angular, React and Vue it should work just fine when it comes to web apps. I would suggest you to take a look into [Deta Space](https://deta.space/). Supports almost all the main stream javascript frameworks. Doesn't support Angular out of the box but should work with custom engine.
And if you are going to try stuff then I would suggest you to join the [discord](https://discord.com/invite/K3gYNqHXUU) server too so that you get faster response if you have any issue.

**Reply by bruno-f-dias** [reply]
Would Space be able to host the Ionic project so that anyone with the URL could access it? 
![image](https://user-images.githubusercontent.com/26816596/214293205-63b9d743-f598-4875-88ce-28eaee8e58ce.png)
